presto-common/pom.xml

```diff
@@ -49,6 +49,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-bytecode</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
We should put limited dependencies to presto-common. What modules need `presto-bytecode` in this case? `presto-bytecode` is a library that can be directly included by dependent modules right?

Reply:
I moved `SqlTypeBytecodeExpression` here from presto-main, so I can use it in other modules. presto-bytecode looked like an independent module with nothing sql related. Do you think its a better fit there?